index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <title>Developer's Summit</title>
+  </head>
+  <body>
+    <!--Navbar-->
+    <nav>
+      <div class="logo-container">
+        <a href="index.html" class="logo">
+          <span class="highlight">Dev</span>Summit<span class="highlight"
+            >.</span
+          >
+        </a>
+      </div>
+      <div class="nav-options">
```

Comment:
- [ ] please you left out the top short bar section with social media links, logout, and language links aligned to the right. please fix that 
your version 👇🏾 
![Screenshot (172)](https://user-images.githubusercontent.com/65298048/174778306-b060e9b3-ac70-42dc-bc39-9c8e5604c603.png)
behance 👇🏾 
![Screenshot (173)_LI](https://user-images.githubusercontent.com/65298048/174778379-485d35d1-3d9e-4541-ba19-9feb1c40deaa.jpg)